cpp-linter performing checkup on a new .c file failed

I am trying to reproduce https://github.com/cpp-linter/cpp-linter/issues/18 in this PR https://github.com/shenxianpeng/test-repo/pull/6

But I have another problem. 

```bash
Performing checkup on p_enemy.c
  INFO:CPP Linter:Running "clang-tidy-14 --export-fixes=clang_tidy_output.yml -checks=boost-* ,bugprone-* ,performance-* ,readability-* ,portability-* ,modernize-* ,clang-analyzer-* ,cppcoreguidelines-* ,-cppcoreguidelines-avoid-magic-numbers ,-readability-magic-numbers --extra-arg= p_enemy.c"
  DEBUG:CPP Linter:Output from clang-tidy:
  
  Traceback (most recent call last):
    File "/home/runner/.local/bin/cpp-linter", line 8, in <module>
      sys.exit(main())
    File "/home/runner/.local/lib/python3.8/site-packages/cpp_linter/run.py", line [101](https://github.com/shenxianpeng/test-repo/actions/runs/3408873516/jobs/5669961676#step:3:106)2, in main
      capture_clang_tools_output(
    File "/home/runner/.local/lib/python3.8/site-packages/cpp_linter/run.py", line 702, in capture_clang_tools_output
      run_clang_format(filename, file, version, style, lines_changed_only)
    File "/home/runner/.local/lib/python3.8/site-packages/cpp_linter/run.py", line 600, in run_clang_format
      for line_range in file_obj["line_filter"][ranges]:
  KeyError: 'line_filter'
  Error: Process completed with exit code 1.
```

For more details see https://github.com/shenxianpeng/test-repo/actions/runs/3408873516/jobs/5669961676

--- a/cpp_linter/run.py
+++ b/cpp_linter/run.py
@@ -597,8 +597,9 @@ def run_clang_format(
     ]
     if lines_changed_only:
         ranges = "diff_chunks" if lines_changed_only == 1 else "lines_added"
-        for line_range in file_obj["line_filter"][ranges]:
-            cmds.append(f"--lines={line_range[0]}:{line_range[1]}")
+        if "line_filter" in file_obj.keys():
+            for line_range in file_obj["line_filter"][ranges]:
+                cmds.append(f"--lines={line_range[0]}:{line_range[1]}")
     cmds.append(PurePath(filename).as_posix())
     logger.info('Running "%s"', " ".join(cmds))
     results = subprocess.run(cmds, capture_output=True)

